addon/components/ember-notify/message.js

```diff
@@ -47,11 +47,21 @@ export default Ember.Component.extend({
   }),
   visibleObserver: Ember.observer('message.visible', function() {
     if (!this.get('message.visible')) {
-      this.send('close');
+      this.send('closeAlias');
     }
   }),
 
   actions: {
+  	closeAlias: function() {
+  		if (this.get('message.closed')) return;
+    	if (this.$().is(':hover')) {
+    		return this.run.later(this, function() {
+    			if (this.get('isDestroyed')) return;
```

Comment:
I did have that together with the check for message.closed at first but decided to leave it there for some reason. Thinking it may have been destroyed by the time the later runs, or something like that. I'll run through the source again to see if I can figure out what I was thinking there.